data.sql

```diff
@@ -48,4 +48,37 @@ update animals set owner_id = 5 where name like 'Charmander';
 update animals set owner_id = 5 where name like 'Squirtle';
 update animals set owner_id = 5 where name like 'Blossom';
 update animals set owner_id = 6 where name like 'Angemon';
-update animals set owner_id = 6 where name like 'Boarmon';
\ No newline at end of file
+update animals set owner_id = 6 where name like 'Boarmon';
+
+insert into vets (name, age, date_of_graduation) values
+  ('William Tatcher', 45, '2000-04-23'), 
+  ('Maisy Smith', 26, '2019-01-17'), 
+  ('Stephanie Mendez', 64, '1981-05-04'), 
```

Comment:
- [ ] Please update the date for `Stephanie Mendez` for `1981-05-17`.

Reply:
hello @arslanbisharat ,I do not understand this required change either. acording to the exercise requirements: 
Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
can you please advise me on how to procced
Artur

Reply:
- I think this is a mix up between the project requirement and code review guideline, but you are good to go on this.👏